src/v8/v8.cc

```diff
@@ -112,6 +113,9 @@ class V8 : public WasmVm {
 
   absl::flat_hash_map<std::string, FuncDataPtr> host_functions_;
   absl::flat_hash_map<std::string, wasm::own<wasm::Func>> module_functions_;
+
+  std::string_view name_section;
+  std::string get_fail_message(std::string function_name, wasm::own<wasm::Trap> trap);
```

Comment:
Style: please move this before fields, ideally right after `getStrippedSource()`. Also, please use `camelCase()`.